bundles/ch.elexis.core.l10n/src/ch/elexis/core/l10n/messages.properties

```diff
@@ -2087,7 +2087,7 @@ ImportLabMapping_titleProblemDialog = Problems
 
 ImporterHost_ExplanationLine1 = This submenu is for preferences of varying import and export
 
-ImporterHost_ExplanationLine2 = modules. Since it is possible that no such modules are installed
+ImporterHost_ExplanationLine2 = Modules. Since it is possible that no such modules are installed
```

Comment:
Isn't this the second part of a longer sentence? If yes, capitalization is not correct here. Additionally, if this is the case, the whole translation should be made differentialy, using placeholders, see https://docs.oracle.com/javase/tutorial/i18n/format/messageFormat.html for example

Reply:
Not sure about it, the german equivalent is this String"Import- und Export-Plugins verwaltet werden."

Reply:
This key is a bit of a mess. But i can revert it if you prefer.

Reply:
I pushed a commit 'Simplyfy ImporterHost explanation'. Please rebase your pull request on the master. Your pull request should no longer contain anything containing ImporterHost_Explanation